azure-quantum/setup.py

```diff
@@ -86,8 +86,8 @@
             "qiskit-qir>=0.1.0b12"
         ],
         "cirq": [
-            "cirq-core==0.14.1",
-            "cirq-ionq==0.14.1",
+            "cirq-core==0.15.0",
```

Comment:
I think this should stay on 0.14.1, as that's the minimal version we know it works with.

Reply:
@anpaz, tests are passing with 0.15.0, what validations do we need to go forward with the update from 0.14.1 to 0.15.0?
Since we're early in the release cycle, it might be a good time to update since we have a month to catch and fix possible issues with the upgrade.

Reply:
Talked about this offline. :)